clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -742,6 +743,11 @@ private void destroyServerStore(ClientDescriptor clientDescriptor, DestroyServer
     boolean isDuplicate = isLifeCycleMessageDuplicate(destroyServerStore);
 
     if (!isDuplicate) {
+      ServerStoreImpl store = ehcacheStateService.getStore(name);
+      if (store != null) {
+        StatisticsManager.nodeFor(store.getPageSource()).clean();
```

Comment:
I guess there is a small chance it could happen.  I added a null check here and also in EhcachePassiveEntity